atoma-service/src/components/openapi.rs

```diff
@@ -16,13 +24,19 @@ pub fn openapi_routes() -> Router {
             (path = CHAT_COMPLETIONS_PATH, api = ChatCompletionsOpenApi),
             (path = EMBEDDINGS_PATH, api = EmbeddingsOpenApi),
             (path = IMAGE_GENERATIONS_PATH, api = ImageGenerationsOpenApi),
+            (path = CONFIDENTIAL_IMAGE_GENERATIONS_PATH, api = ConfidentialImageGenerationsOpenApi),
+            (path = CONFIDENTIAL_EMBEDDINGS_PATH, api = ConfidentialEmbeddingsOpenApi),
+            (path = CONFIDENTIAL_CHAT_COMPLETIONS_PATH, api = ConfidentialChatCompletionsOpenApi),
         ),
         tags(
             (name = "health", description = "Health check"),
             (name = "metrics", description = "Metrics"),
             (name = "chat", description = "Chat completions"),
             (name = "embeddings", description = "Embeddings"),
             (name = "images", description = "Image generations"),
+            (name = "confidential-images", description = "Confidential image generations"),
```

Comment:
we are not actually using the tags. Also we should use them, but not like this. Doesn't make any sense tag equal function. They should be used properly, grouping similar function. But it doesn't have to be part of this PR.